src/Components/Search/Previews/Grids/ArtistSearch/RelatedArtworks.tsx

```diff
@@ -19,7 +20,7 @@ export const RelatedArtworksPreview: React.SFC<RelatedArtworksPreviewProps> = ({
   ).map(x => x.node)
 
   const relatedArtworks = artworks.map((artwork, i) => (
-    <Box width="50%" key={i}>
+    <Box width={["0%", "100%", "100%", "50%"]} key={i}>
```

Comment:
Does it still render at 0?

Reply:
It does not - for this PR I'm suppressing previews altogether. As noted, this is naive because there actually are cases where previews should show up in the comps, but ya gotta start somewhere.

This property requires a value for XS, so I put 0% there for now. We'll revisit this once we improve the UX for XS.

Reply:
Aight. Mostly was just a question out of curiosity, so thanks 👍